tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/appsettings.json

```diff
@@ -133,7 +133,8 @@
     "GraphUrl": "https://graph.microsoft.com",
     "IsBackwardCompatible": false,
     "IsClientMappedWithHierachy": false,
-    "MatterUsersGroup": "Matter Center Users"
+    "MatterUsersGroup": "Matter Center Users",
+    "OverwriteDupliacteFileNameWithDateTimeFor": "Email Only"
```

Comment:
Added a new proeprty called OverwriteDupliacteFileNameWithDateTimeFor and the values for this one can be Email Only, Document Only or Both